content/en/faqs/Is-Privacy-Badger-breaking-YouTube.md

```diff
@@ -0,0 +1,10 @@
+---
+question: How should I handle issues with Privacy Badger on YouTube? 
+weight: 280
+---
+
+Issues with YouTube while Privacy Badger is enabled are rarely caused by Privacy Badger. However, try temporarily disabling Privacy Badger for YouTube and check if that resolves the issue: Navigate to YouTube, click the Privacy Badger icon in your browser toolbar, and click "Disable for this site" in the popup.
```

Comment:
Let's just link "try disabling Privacy Badger on YouTube" to `#I-found-a-bug%21-What-do-I-do-now` instead of repeating these instructions here.

Reply:
Could then maybe make this entry two paragraphs instead of three.

Reply:
good call, done!